<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthDialog</class>
 <widget class="QDialog" name="AuthDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>236</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Authentification</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>301</width>
     <height>27</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>12345-abcde</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="secretEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>301</width>
     <height>27</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>tai-a1b2c3d4e5</string>
   </property>
  </widget>
  <widget class="QLabel" name="keyLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>66</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Clé API</string>
   </property>
  </widget>
  <widget class="QLabel" name="secretLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>66</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Secret</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="saveIndentsCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Sauvegarder les identifiants dans un cache sécurisé 
( nécessite un mot de passe global QGIS)</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorMessage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>224</red>
         <green>27</green>
         <blue>36</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>224</red>
         <green>27</green>
         <blue>36</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>190</red>
         <green>190</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AuthDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AuthDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
